.eslintignore

```diff
@@ -0,0 +1 @@
+types
```

Comment:
Instead of doing this file, let's use the `ignorePatterns` in `eslintrc.js` instead. https://eslint.org/docs/user-guide/configuring/ignoring-code#the-eslintignore-file